Lightrun Integration - External task driver

amit-shuster opened the pull request.

hashicorp-cla commented:
[![CLA assistant check](https://cla.hashicorp.com/pull/badge/not_signed)](https://cla.hashicorp.com/hashicorp/nomad?pullRequest=11203)


Thank you for your submission! We require that all contributors sign our Contributor License Agreement ("CLA") before we can accept the contribution. [Read and sign the agreement](https://cla.hashicorp.com/hashicorp/nomad?pullRequest=11203)

[Learn more about why HashiCorp requires a CLA and what the CLA includes](https://www.hashicorp.com/cla)


---
**Amit Shuster** seems not to be a GitHub user.
You need a GitHub account to be able to sign the CLA. If you already have a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).

<sub>Have you signed the CLA already but the status is still pending? [Recheck it](https://cla.hashicorp.com/check/hashicorp/nomad?pullRequest=11203).</sub>

lgfa29 reviewed (changes requested):
Thank you for the PR @amit-shuster and for creating a new task driver for Nomad 🙂 

I pushed a small commit to fix an error in the website navigation and to also format the Markdown doc file a bit to make it look more like our other plugin pages.

A few notes:

1) would you mind expanding on what [these configuration](https://github.com/hashicorp/nomad/pull/11203/files#diff-db3e3967ad55e63727ee7b1e779ab37d4c9e50ede9d29777e0d369b0024bfd73R62-R64) do? They seem straight forward, but I don't know enough about Lightrun to attempt writing them myself 😅 

2) The original text had this paragraph, but I don't know what `example/example.driver.nomad` it's referring to, so I remove it. Was it supposed to link somewhere?
    > A full job file example can be found at `example/example.driver.nomad`

3) Would you mind signing the CLA so we can get these merged?

Thank you.

amit-shuster commented:
Hi @lgfa29, 
I addressed all of your comments, please let me know if there are any additional changes require.
 Regarding the SLA, I did sign it about a week ago, but had some issue with my GitHub account, as you can see now there are two "Amit Shuster" on this PR, can you fix it on your side or do you need me to open a new PR?
 
 Thank you,
Amit Shuster

lgfa29 commented:
> Regarding the SLA, I did sign it about a week ago, but had some issue with my GitHub account, as you can see now there are two "Amit Shuster" on this PR, can you fix it on your side or do you need me to open a new PR?

Ah I see the problem. Your first two commits had the wrong email address, which is not associated with your GitHub account. I would have to `git push --force` to your branch to fix them, would that be OK?
